common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -81,10 +83,13 @@ object LogKey extends Enumeration {
   val REDUCE_ID = Value
   val REMOTE_ADDRESS = Value
   val RETRY_COUNT = Value
+  val RETRY_INTERVAL_MILLISECOND = Value
```

Comment:
As per https://github.com/apache/spark/pull/45903/, the default unit will be MILLISECOND. We can ignore it in the name.

Reply:
Okay, This is what i want, done.